src/hotspot/share/opto/escape.cpp

```diff
@@ -4830,7 +4873,27 @@ void ConnectionGraph::split_unique_types(GrowableArray<Node *>  &alloc_worklist,
         }
         mem = mem->in(MemNode::Memory);
       }
+      if (mem->Opcode() == Op_NarrowMemProj) {
+        const TypePtr* at = mem->adr_type();
+        uint idx = (uint) _compile->get_alias_index(at->is_ptr());
```

Comment:
```suggestion
        uint alias_idx = (uint) _compile->get_alias_index(at->is_ptr());
```
for consistency with the other code

Reply:
Well, actually it is already being used inconsistently... hmm. More expressive may still be better. Up to you.